src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -162,6 +166,34 @@ internal static void ThrowIfUnsupportedType()
         [CLSCompliant(false)]
         public Vector128<ulong> AsUInt64() => As<ulong>();
 
+        /// <summary>Determines whether the specified <see cref="Vector128{T}" /> is equal to the current instance.</summary>
+        /// <param name="other">The <see cref="Vector128{T}" /> to compare with the current instance.</param>
+        /// <returns><c>true</c> if <paramref name="other" /> is equal to the current instance; otherwise, <c>false</c>.</returns>
+        /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
+        public bool Equals(Vector128<T> other)
```

Comment:
This can be optimized by `Sse41` and `Avx` intrinsic.

Reply:
I want to get the CoreFX side changes up first so I can get test in before enabling intrinsic support.